.github/workflows/service_test_redis.yml

```diff
@@ -192,3 +192,26 @@ jobs:
           OPENDAL_REDIS_ENDPOINT: tcp://127.0.0.1:6379
           OPENDAL_REDIS_ROOT: /
           OPENDAL_REDIS_DB: 0
+
+  binding_java_redis:
```

Comment:
Is it possible to make it a matrix so that we can test core, binding/java, bindings/python, ...?

Reply:
> Is it possible to make it a matrix so that we can test core, binding/java, bindings/python, ...?

It's possible but I'm afraid that we need a lot of if-else to handle the setup and teardown logics for different languages.

Perhaps we can factor out the service setup as a reuseable workflow/action and refer them in each specific language's job.

Reply:
Ok, let's split this into a new issue instead.